mapdata/WarcraftLegacies/UnitData/Skin/909127797.json

```diff
@@ -53,7 +53,7 @@
     {
       "Id": 1651864693,
       "Type": 3,
-      "Value": "Summoning site for the Void demons of the Burning Legion.|r|n|cff99b4d1Can only build 2 before the Legion has been summoned. 12 post-summon.|n|cfff5962dTrains:|R Eredar Summoner, Succubus|n|cfff5962dResearches:|R Eredar Summoner Training, Succubus Training"
+      "Value": "Summoning site for the Void demons of the Burning Legion.|r|n|cff99b4d1Can only build 3 before the Legion has been summoned. 12 post-summon.|n|cfff5962dTrains:|R Eredar Summoner, Succubus|n|cfff5962dResearches:|R Eredar Summoner Training, Succubus Training"
```

Comment:
This isn't really related to this PR but can we just remove the word "Void" here? These aren't void related.

Reply:
Yeah sure!